bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -22,7 +23,15 @@
 			<channel id="videoUrl" typeId="url">
 				<label>Video URL</label>
 			</channel>
+			<channel id="runState" typeId="runState">
+				<label>Run State</label>
```

Comment:
This label is redundant as it's the same as defined in the channel type.

Reply:
Fixed